datahub-web-react/src/conf/theme/hmrc.config.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
This file is a copy of DataHub's `theme_light.config.json` but with the `primary-color`, `logoUrl`, `title` and `homepageMessage` changed to use CIP branding